charts/aws-efs-csi-driver/templates/controller-pdb.yaml

```
@@ -0,0 +1,24 @@
+{{- if and .Values.controller.podDisruptionBudget.enabled (not .Values.nodeComponentOnly) -}}
```

```suggestion
{{- if .Values.controller.podDisruptionBudget.enabled -}}
```

`.Values.nodeComponentOnly` is not a flag/feature of this chart, this only exists on the EBS Controller